src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -91,20 +105,29 @@ public void renewClusterSlots(Connection jedis) {
           }
         }
 
+        // Then, we use startNodes to try, as long as startNodes is available,
+        // whether it is vip, domain, or physical ip, it will succeed.
+        ArrayList<HostAndPort> startNodeList = new ArrayList<>(startNodes);
```

Comment:
`startNodes` can be null. So we should do a safety check.

Reply:
👍，done